test/functional/http/get/accounts/accounts.js

```diff
@@ -393,6 +394,26 @@ describe('GET /accounts', () => {
 				});
 		});
 
+		it('should return correct delegate approval for a delegate account', async () => {
+			const promises = [
+				constantsEndPoint.makeRequest(),
+				accountsEndpoint.makeRequest(
+					{ address: accountFixtures.existingDelegate.address },
+					200
+				),
+			];
+
+			const [
+				{ body: { data: constansts } },
+				{ body: { data: [{ delegate }] } },
+			] = await Promise.all(promises);
+			const calculatedApproval = parseFloat(
```

Comment:
we shouldn't be creating `calculatedApproval` function here, rather we should make use of the `calculatedApproval` function from the module and pass the object and expect the same behavior. If in the case in future someone change the logic, the test `calculatedApproval` needs to be updated as well.